sdk/core/src/services/github/client/mod.rs

```diff
@@ -32,50 +32,61 @@ impl Client {
         format!("{GH_URL}/repos/{repo_name}/commits/{default_branch}")
     }
 
+    /// Creates the URL one must use in an http request for
+    /// acquiring the latest commit hash from a given branch
+    fn get_latest_release_url(&self, repo: &Repo) -> String {
```

Comment:
Since this is an option in theory it could be `None`, correct?  What happens if it is `None`.

Reply:
Good point.  I looked into the origin and necessity of the field and it _must_ be filled in or we don't have any way to get the repo information because it's used to build urls for other http calls.  The full_name comes from the initial call to extract repo information from the github organization, so I simply chnaged the `Option<String>` to a String.